### ISSUE TITLE
Visible label and accessible name do not match on Pages

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6786]
### Descriptive summary

On the `Pages` page, the label of the `Bold` dropdown menu and the accessible name don't match and therefore fail WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin user, go to https://dev.nurax.samvera.org/pages/edit?locale=en and run the SiteImprove browser extensions

![visible_label_no_matching](https://github.com/samvera/hyrax/assets/32644254/ad0ba8a5-72a3-4f83-aa6d-74afc6edfd80)

**Comment by christhepianist** [comment]
Tested and confirmed this issue.

**Comment by jlhardes** [comment]
Also exists Dashboard-wide in the case of having an unread Notification (see attached screenshot). Probably similar code change needed but in different places.

![Image](https://github.com/user-attachments/assets/b23889d7-23be-4e80-b791-7e9e55a8a242)

**Comment by jlhardes** [comment]
TinyMCE 7 does not have this accessibility issue even in the basic version and appears to match up the drop-down format label with the name of the selected format (Heading 3 in attached tinymce7.png screenshot). I think an update on TinyMCE is needed to fix the originally reported accessibility issue when editing Pages in the Dashboard.

![Image](https://github.com/user-attachments/assets/7aa94b49-34d8-4ac8-9dac-5f438ad35afe)

**Comment by rjkati** [comment]
Thanks for looking into this, @jlhardes. I will close this ticket and create another one to update TinyMCE
